pkg/kubelet/kubelet.go

```diff
@@ -1595,7 +1595,7 @@ func (kl *Kubelet) syncPod(o syncPodOptions) error {
 		// under the qos cgroup hierarchy.
 		// Check if this is the pod's first sync
 		firstSync := true
-		for _, containerStatus := range apiPodStatus.ContainerStatuses {
+		for _, containerStatus := range append(apiPodStatus.InitContainerStatuses, apiPodStatus.ContainerStatuses...) {
```

Comment:
Is it possible that a initContainer could be running, but the pod would still need to be synced? I.e. can the syncing still need to happen if one of the initContainers initially failed?

Reply:
The purpose of the code beginning at line 1593 is to apply resource parameters for the pod.
The application of resource parameters should affect init containers as well.

Reply:
Cool - could I trouble you to add that to your commit message or pr body?